templates/vault.systemd.erb

```diff
@@ -8,11 +8,16 @@ After=basic.target network.target
 User=<%= scope.lookupvar('vault::user') %>
 Group=<%= scope.lookupvar('vault::group') %>
 ExecStart=<%= scope.lookupvar('vault::bin_dir').shellescape %>/vault server \
-  -config <%= scope.lookupvar('vault::config_dir').shellescape %> <%= scope.lookupvar('vault::extra_options').shelljoin %>
+-config <%=
```

Comment:
Should this be indented, similar to the line it replaces?

Reply:
You're right, I should indent for readability. Fixed in the next commit.